<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNavigator</class>
 <widget class="QTabWidget" name="HNavigator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>665</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>208</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Histogram Navigator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/New Canvas button icon</normaloff>:/New Canvas button icon</iconset>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tabInputFile">
   <attribute name="title">
    <string>Input File</string>
   </attribute>
   <widget class="QComboBox" name="hNavigatorInputFileCB">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>335</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="hNavigatorCanvasGB">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>480</y>
      <width>336</width>
      <height>157</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>213</red>
          <green>253</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>156</red>
          <green>194</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>213</red>
          <green>253</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>156</red>
          <green>194</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>106</red>
          <green>104</green>
          <blue>100</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>156</red>
          <green>194</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>156</red>
          <green>194</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="title">
     <string>Controls</string>
    </property>
    <widget class="QComboBox" name="hCanvasCB">
     <property name="geometry">
      <rect>
       <x>108</x>
       <y>114</y>
       <width>39</width>
       <height>18</height>
      </rect>
     </property>
     <property name="toolTip">
      <string comment="Disambiguation" extracomment="Canvas number to display"/>
     </property>
     <property name="statusTip">
      <string extracomment="Status tip"/>
     </property>
     <property name="whatsThis">
      <string extracomment="whatsThis"/>
     </property>
    </widget>
    <widget class="QLabel" name="currentCanvasLB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>114</y>
       <width>93</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current canvas</string>
     </property>
    </widget>
    <widget class="QComboBox" name="canvasSizeCB">
     <property name="geometry">
      <rect>
       <x>244</x>
       <y>114</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="canvasSizeLB">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>116</y>
       <width>75</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Canvas size</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="displayControlsTB">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>16</y>
       <width>325</width>
       <height>95</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>255</green>
           <blue>202</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>197</green>
           <blue>131</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>197</green>
           <blue>131</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>212</red>
           <green>208</green>
           <blue>200</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>212</red>
           <green>208</green>
           <blue>200</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>212</red>
           <green>208</green>
           <blue>200</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="actionsTP">
      <attribute name="title">
       <string>Actions</string>
      </attribute>
      <widget class="QPushButton" name="unZoomPB">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>6</y>
         <width>59</width>
         <height>18</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; text-decoration: underline; color:#0000ff;&quot;&gt;Unzoom&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Resets the X and Y axis to their default for all currently selected histograms&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Unzoom</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="statFitTP">
      <attribute name="title">
       <string>Stat and Fit</string>
      </attribute>
      <widget class="QCheckBox" name="fitCB">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>26</y>
         <width>169</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Display fit results panel</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="statCB">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>6</y>
         <width>231</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Display histogram statistics panel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="twoDPlotsTP">
      <attribute name="title">
       <string>2D plots</string>
      </attribute>
      <widget class="QGroupBox" name="twoDGB">
       <property name="geometry">
        <rect>
         <x>-2</x>
         <y>-2</y>
         <width>325</width>
         <height>75</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QCheckBox" name="legoCB">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>26</y>
          <width>73</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Lego</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="surfCB">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>4</y>
          <width>73</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Surface</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="contourCB">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>4</y>
          <width>73</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Contour</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="COLZCB">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>26</y>
          <width>73</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>COLZ</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="logxCB">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>4</y>
          <width>73</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Log X</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="logyCB">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>26</y>
          <width>73</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Log Y</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="saveComponentsPB">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>136</y>
       <width>213</width>
       <height>18</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>117</red>
           <green>201</green>
           <blue>122</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>117</red>
           <green>201</green>
           <blue>122</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>117</red>
           <green>201</green>
           <blue>122</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>Save selected components to file</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="flattenHierarchyCB">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>136</y>
       <width>63</width>
       <height>19</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Flatten out hierarchy">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; text-decoration: underline; color:#0000ff;&quot;&gt;Flatten&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If checked, all selected histograms are&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;saved in the file in the top level directory,&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;without preserving the original folder structure.&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Flatten</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="hNavigatorTreeFrame">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>26</y>
      <width>336</width>
      <height>453</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>212</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>212</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>208</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tabOutputFile">
   <attribute name="title">
    <string>Output File</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
